src/main/java/eu/dissco/core/digitalspecimenprocessor/service/AnnotationPublisherService.java

```diff
@@ -129,7 +133,8 @@ private List<AnnotationProcessingRequest> convertJsonPatchToAnnotations(
         .getOdsSourceSystemName();
     for (JsonNode action : jsonNode) {
```

Comment:
Looked at this but it is tricky as the format of the action is differently. It would require more parameters, and I wasn't convinced the extra complexity in the addAddOperation would be worth it.

Reply:
:+1: